gini/gini-vision-lib-ios
# New onboarding images, texts and an additional page for iPad

Added new onboarding page and updated old images and texts.

### How to test
From the camera screen press "Hilfe" button and see that the new images and texts appear. Also on iPad see that a new page is added.

### Merging
Automatic.

### Ticket 
Issues #90 .
